data/json/monsters.json

```diff
@@ -1971,6 +1986,7 @@
     "armor_bash":6,
     "armor_cut":6,
     "vision_day":50,
+    "vision_night":5,
```

Comment:
Uh, these are only a fast and deadly-in-melee critter as is, what with the small size, armor, and speed.  If you're going "canid NV" that's not out of the question, but nevertheless this is gonna justify complaints.